guide/getting-started/common-user-flows.md

```diff
@@ -241,11 +240,11 @@ Equivalent to creating an invoice, requesting bitcoin involves entering informat
    height = 336
 %}
 
-While it is initially intuitive to re-use the same receiving address over and over, this practice is highly discouraged. Anyone with this address can easily see what other payments are being made to it and how the bitcoin are spent. The UI should encourage users to create new addresses for each invoice. This also makes it easier for the software to understand when a specific payment has been received.
+While it is initially intuitive to re-use the same receiving address repeatedly, this practice is highly discouraged. Anyone with this address can easily see what other payments are being made and how the bitcoin is spent. The UI should encourage users to create new addresses for each invoice. This also makes it easier for the software to understand when a specific payment has been received.
```

Comment:
> how the bitcoin is spent

Shouldn't it be plural as it's a quantity? "How the Euro is spent" also doesn't sound right.

Reply:
Fixed in https://github.com/BitcoinDesign/Guide/commit/8796246fd2cb09bb9065848f6f28145b3566637d